core/arch/arm/kernel/core_bootcfg.c

```diff
@@ -169,7 +169,7 @@ static struct map_area bootcfg_memory_map[] = {
 };
 
 /*
- * bootcfg_get_pbuf_is_handler - return the platform specfic pbuf_is
+ * bootcfg_get_pbuf_is_handler - return the platform specific pbuf_is
```

Comment:
What about changing the flag to `CFG_GENERIC_BOOT` and putting `generic` in the filenames?
I would rather not have `generic` in the file names, but I don't have a strong opinion on that.